README.rst

```diff
@@ -90,6 +90,25 @@ And here is a sparse adjacency matrix:
     from scipy.sparse import random
     adj_sparse = random(100, 100, density=0.1)
     dgm = ripser_parallel(adj_sparse, metric="precomputed", maxdim=2, n_threads=-1)
+
+Edge Collapser
+--------------
+
+Push the computation to higher homology dimensions and larger point clouds/distance matrices/adjacency matrices using edge collapses:
+
+.. code-block:: python
+
+    dgm_higher = ripser_parallel(pc, maxdim=5, collapse_edges=True, n_threads=-1)
+
+Weighted Rips Filtrations
+-------------------------
+
+Use the ``weights`` parameter to constructed a weighted Rips filtration as defined in `this paper <https://doi.org/10.1007/978-3-030-43408-3_2>`_. You can either pass a custom 1D array of vertex weights, or the string ``"DTM"`` (together with ``weight_params``) for distance-to-measure reweighting:
+
+.. code-block:: python
+
+    dgm_dtm = ripser_parallel(pc, weights="DTM", n_threads=-1)
+
     
```

Comment:
You are saying that users haven't read where ``pc`` is defined above?  Personally, I think this is fine as this is a very short "basic usage" section.  Btw, the same criticism could be applied to the edge collapser section just above, and the metric space example above that, no?

Reply:
Oh no, what I mean in this particular case, because I am not a user of `weights` feature is can just the user run `ripser_parallel` with the parameter `weights="DTM"` like in the example ? Because, it mentions `weight_params` and I thought that you need a combination of `weights` and `weight_params` in order to make things work.

It is just my ignorance that maybe confuses me

Reply:
Ah I see, yes I can explain ``weight_params`` better, let me try something else.

Reply:
Done, let me know!

Reply:
Yes ! I think that the user has now more indication if he want to make it work.
And on the meantime the notebook example can be made, the text will help users of the feature

Reply:
I agree!